src/tests/activity.test.js

```diff
@@ -1 +1,141 @@
-test.skip('skip', () => {});
+import request from 'supertest';
+import app from '../app';
+import activity from '../models/activity';
+import activitySeed from '../seeds/activity';
+
+const mockActivity = {
+  name: 'Boxing',
+  description: 'vestibulum aliquet ultrices erat tortor sollicitudin mi non mi integer ac neque duis',
+  isActive: false,
+};
+
+const mockActivityExistingName = {
+  name: 'Chandal',
+  description: 'Any description',
+  isActive: false,
+};
+
+const mockLessName = {
+  name: '',
+  description: 'Some text',
+  isActive: true,
+};
+
+const mockLessDescription = {
+  name: 'Name',
+  description: '',
+  isActive: true,
+};
+
+const mockLessIsActive = {
+  name: 'Names',
+  description: 'some description',
+  isActive: '',
+};
+
+beforeAll(async () => {
```

Comment:
Maybe we should try this Instead of using beforeAll(), to avoid lines 132 to 140
```
beforeEach(async () => {
  await Activity.collection.insertMany(activitySeed);
});

afterEach(async () => {
  await Activity.collection.deleteMany();
});
```

Reply:
fixed